lib/hijack.rb

```diff
@@ -29,6 +29,7 @@ def hijack(info: nil, &blk)
         Scheduler::Defer.later(
           "hijack #{params["controller"]} #{params["action"]} #{info}",
           force: false,
+          current_user: current_user&.id,
```

Comment:
Just for my own understanding, anon defer queue requests would all be grouped together, under `current_user: nil`, right?

Reply:
Correct.